Mathlib/Data/Nat/PrimeFin.lean

```diff
@@ -22,6 +23,12 @@ theorem infinite_setOf_prime : { p | Prime p }.Infinite :=
   Set.infinite_of_not_bddAbove not_bddAbove_setOf_prime
 #align nat.infinite_set_of_prime Nat.infinite_setOf_prime
 
+instance Primes.infinite : Infinite Primes := infinite_setOf_prime.to_subtype
+
+open scoped Cardinal
+
+theorem Primes.cardinal_mk_eq_aleph0 : #Primes = ℵ₀ := Cardinal.mk_eq_aleph0 Primes
```

Comment:
I'm not really in favour of adding this: I doubt it will ever be used and it forces us to import `Mathlib.SetTheory.Cardinal.Basic` into this file.

The mathematical content is the pair of instances `Countable Primes` and `Infinite Primes` (which are definitely worth having) but I'm not convinced about encoding this using cardinals.